compiler/codegen/mli.cpp

```diff
@@ -920,16 +887,16 @@ std::string MLIContext::genMemCleanup(Type* t, const char* var) {
 }
 
 std::string MLIContext::genMarshalCall(const char* skt, const char* var,
-                                       Type* t, bool out) {
+                                       Type* t, bool push) {
   std::string gen;
   int64_t id = this->assignUniqueTypeID(t);
 
-  gen += out ? marshal_push_prefix : marshal_pull_prefix;
+  gen += push ? "chpl_mli_push_" : "chpl_mli_pull_";
```

Comment:
It looks like there's still opportunities to save these strings as globals and reuse them, what caused you to move away from it?

Reply:
They were only being used in one place, so my preference was to do away with the global and just use a string.

Reply:
But now they're being used in 4+ places, so far as I can tell

Reply:
I'll go ahead and add some global strings for them.